mlrun/runtimes/pod.py

```diff
@@ -874,6 +876,145 @@ def with_requests(
         """
         self.spec.with_requests(mem, cpu, patch=patch)
 
+    @staticmethod
+    def detect_preemptible_node_selector(node_selector: dict[str, str]) -> list[str]:
+        """
+        Check whether any provided node selector matches preemptible selectors.
+
+        Args:
+            node_selector: User-provided node selector mapping.
+
+        Returns:
+            List of `"key='value'"` strings that match a preemptible selector.
+        """
```

Comment:
The rest of the docstrings in this file are in reST format - we should keep everything aligned at least.

Reply:
We decided in committee that moving forward, all new functions will use the new documentation format, and we'll find a way to align the entire codebase later. I've asked Adam to update this in the relevant channels so we're all aligned.